sourdough/resources/views/login.html

```diff
@@ -0,0 +1,47 @@
+<form class="auth-form" name="login" action="javascript:void(0)" method="post">
+    <div class="auth-header">Sign In</div>
+    <div class="labkey-error" id="errors"></div>
+    <div class="auth-form-body" style="background-size: 320px;background-image: url('https://thumbs.dreamstime.com/b/sourdough-freshly-baked-bread-as-texture-background-sourdough-freshly-baked-bread-white-background-157653690.jpg');">
+        <label for="email">Email</label>
+        <input id="email" name="email" type="text" class="input-block" tabindex="1" autocomplete="off">
+        <label for="password">Password</label>
+        <div class="forgot-password-link">
+            <a href="login-resetPassword.view?">Forgot password</a>
+        </div>
+        <input id="password" name="password" type="password" class="input-block" tabindex="2" autocomplete="off">
+        <input tabindex="3" type="checkbox" name="remember" id="remember"  checked> Remember my email address
+        <div class="termsOfUseSection" hidden>
+            <div class="auth-header auth-item">Terms of Use</div>
+            <div class="toucontent auth-item termsOfUseContent"></div>
+            <div class="auth-item">
+                <input type="checkbox" tabindex="4" name="approvedTermsOfUse" id="approvedTermsOfUse" class="auth-item" unchecked>
+                <label for="approvedTermsOfUse">I agree to these terms</label>
+            </div>
+        </div>
+        <input type="hidden" name="termsOfUseType" id="termsOfUseType">
+        <div class="auth-item auth-credentials-submit">
+            <input type="submit" tabindex="-1" class="loginSubmitButton"/>
+            <a tabindex="5" class="labkey-button primary signin-btn"><span>Sign In</span></a>
+            <span class="registrationSection" hidden>
+                <a class="labkey-button" id="registerButton" href="login-register.view?">Register</a>
+            </span>
+        </div>
+        <div class="signing-in-msg" hidden>
+            <span class="fa fa-spinner fa-pulse"></span> Signing you in. This may take a moment.
+        </div>
+
+        <input type="hidden" name="returnUrl">
+        <input type="hidden" id="urlhash" name="urlhash">
+        <input type="hidden" name='X-LABKEY-CSRF'>
+    </div>
+</form>
+<div class="otherLoginMechanismsSection" hidden>
+    <div class="auth-header">Or Sign In using</div>
+    <div class="otherLoginMechanismsContent"></div>
+</div>
```

Comment:
This was likely mostly just copied from our existing login page html right? I'm just wondering if we should simplify and remove as much as possible here of the "extra" stuff like this other login mechanism section since it is a bit internal and something we could theoretically change in a backwards breaking way in the future.

Reply:
Good call! Removed